### PR TITLE
Migrate to FW SDK v6.0.0

### PR DESCRIPTION
**Pull request by mohsenoid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,15 +7,16 @@ This repo holds the Firework Android SDK example applications.
...
+* [Jetpack Compose](#jetpack-compose-example-app)
 
 **Read more about the integration here** https://docs.firework.tv/
 
 ---
 
-## View Options example app
+#### View Options example app
 
 This example app demonstrates all possible view options that can be used for the `VideoFeedView` initialization.
```
Comment (jrodriguezmontoya) [inline README.md:21]:
```suggestion
This example app demonstrates all possible view options that can be used for the `FwVideoFeedView` initialization.
```

File: README.md [hunk 2]
```diff
@@ -33,20 +34,20 @@ This example app demonstrates all possible view options that can be used for the
 
 In this example app, the `VideoFeedView` is initialized with different feedResources: Discovery, Playlist, Channel, Dynamic Content.
```
Comment (jrodriguezmontoya) [inline README.md:35]:
```suggestion
In this example app, the `FwVideoFeedView` is initialized with different feedResources: Discovery, Playlist, Channel, Dynamic Content.
```

Comment (jrodriguezmontoya) [inline README.md:35]:
```suggestion
In this example app, the `FwVideoFeedView` is initialized with different feedResources: Discovery, Playlist, Channel, Dynamic Content.
```

File: README.md [hunk 3]
```diff
@@ -106,3 +97,26 @@ It contains two activities: which .
 2. `ShareLinkActivity` - handles the deep link and opens the shared video.
 Also this example demonstrates how to replace the base url of the share video with custom one.
 
+---
+## Picture-in-picture example app
+
+In this example app, the `VideoFeedView` is initialized for switch to picture-in-picture mode if user in in player and tries to leave the app, or when user presses the down shevrun on player, or when related public API is being called.
```
Comment (jrodriguezmontoya) [inline README.md:103]:
```suggestion
In this example app, the `FwVideoFeedView` is initialized for switch to picture-in-picture mode if user in in player and tries to leave the app, or when user presses the down chevron on player, or when related public API is being called.
```

File: README.md [hunk 4]
```diff
@@ -106,3 +97,26 @@ It contains two activities: which .
...
+
+[Picture-in-picture example app](picture_in_picture )
+
+| PiP Feed                                              | Player screen in PiP mode                                    |
```
Comment (jrodriguezmontoya) [inline README.md:107]:
```suggestion
| PIP Feed                                              | Player screen in PIP mode                                    |
```

File: README.md [hunk 5]
```diff
@@ -106,3 +97,26 @@ It contains two activities: which .
...
+| ----------------------------------------------------- | ------------------------------------------------------------ |
+| ![pip Screenshot](picture_in_picture/Screenshot1.png) | ![player in pip mode Screenshot](picture_in_picture/Screenshot2.png) |
+
+---
+## Jetpack Compose example app
+
+In this example app, the `VideoFeedView` is initialized for showing a discovery feed using Jetpack Compose.
```
Comment (jrodriguezmontoya) [inline README.md:114]:
```suggestion
In this example app, the `FwVideoFeedView` is initialized for showing a discovery feed using Jetpack Compose.
```

File: README.md (hunk 3 above)
Comment (jrodriguezmontoya) [inline README.md:103]:
```suggestion
In this example app, the `FwVideoFeedView` is initialized for switch to picture-in-picture mode if user in in player and tries to leave the app, or when user presses the down chevron on player, or when related public API is being called.
```

File: picture_in_picture/app/src/main/java/com/firework/example/pictureinpicture/ExampleApplication.kt [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+        // build Firework Android SDK v2 configuration
+        val config = FireworkSdkConfig.Builder(this)
+            .checksumRequired(false)
+            .clientId(FW_CLIENT_ID) // Client OAUTH Id
+            .userId("example app user ID") // User Id in your eco-system
+            .imageLoader(GlideImageLoaderFactory.createInstance()) // glide, picasso, or your implementation
+            .build()
+
+        // initialize Firework Android SDK v2
```
Comment (jrodriguezmontoya) [inline picture_in_picture/app/src/main/java/com/firework/example/pictureinpicture/ExampleApplication.kt:23]:
```suggestion
        // initialize Firework Android SDK v6
```

File: picture_in_picture/app/build.gradle.kts [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+        versionCode = 1
+        versionName = "1.0.0"
+    }
+
+    buildTypes {
+        defaultConfig {
+            buildConfigField("String", "FW_CLIENT_ID", "\"f6d6ec1275217f178cdff91363825cb390e038c1168f64f6efa23cb95ec6b325\"")
+            buildConfigField("String", "FW_CHANNEL_ID", "\"7RXwK8k\"")
+            buildConfigField("String", "FW_PLAYLIST_ID", "\"g4lA0g\"")
```
Comment (jrodriguezmontoya) [inline picture_in_picture/app/build.gradle.kts:27]: These values are not beign used in the sample app
```suggestion
```

File: picture_in_picture/app/src/main/java/com/firework/example/pictureinpicture/ExampleApplication.kt (hunk 6 above)
Comment (mohsenoid) [reply picture_in_picture/app/src/main/java/com/firework/example/pictureinpicture/ExampleApplication.kt:23]: Fixed

File: picture_in_picture/app/build.gradle.kts (hunk 7 above)
Comment (mohsenoid) [reply picture_in_picture/app/build.gradle.kts:27]: Copy/Paster effect, removed

File: picture_in_picture/app/src/main/res/layout/activity_main.xml [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/pip"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/enter_pip_mode"
+        app:layout_constraintBottom_toTopOf="@id/details"
+        app:layout_constraintTop_toBottomOf="@id/fwVideoFeedView" />
+
```
[the remark this answers is not in the document]
Comment (mohsenoid) [reply picture_in_picture/app/src/main/res/layout/activity_main.xml:23]: Yeah, it makes sense. I will do that.

File: view_options/app/src/main/AndroidManifest.xml [hunk 9]
```diff
@@ -1,23 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
```
Comment (jrodriguezmontoya) [inline view_options/app/src/main/AndroidManifest.xml:4]: ❓ Do we need this permission?

File: single_host_livestream/app/src/main/AndroidManifest.xml [hunk 10]
```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
```
[the remark this answers is not in the document]
Comment (mohsenoid) [reply single_host_livestream/app/src/main/AndroidManifest.xml:4]: Yeah, something in the latest Android SDK which causes Lint error.

**Review by jrodriguezmontoya — APPROVED** [review]
Added some suggestions, but it LGTM

**Pull request merged**